src/_P095_ILI9341.ino

```diff
@@ -9,6 +9,8 @@
 #define PLUGIN_VALUENAME1_095 "TFT"
 #define PLUGIN_095_MAX_DISPLAY 1
 
+#define PLUGIN_FONT_MODE      // enable to use fonts in this plugin
```

Comment:
This define is unconditional, so you cannot exclude it via the build configuration to make smaller builds.
Also the define doesn't have the plugin number in its name, so that makes it a bit too generic.

So better change it into something like this:
```c++
#include "_Plugin_Helper.h"

#if !defined(LIMIT_BUILD_SIZE) && !defined(PLUGIN_095_FONT_INCLUDED)
#define PLUGIN_095_FONT_INCLUDED
#endif
```

And later also change the check for this define.

Reply:
done your proposed naming conventions.